doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -250,6 +251,27 @@ The following is an example of a record with non-trivial subtyping relation:
    E[Γ] ⊢ \mathsf{packType}@\{i\} =_{βδιζη}
    \mathsf{packType}@\{j\}~\mbox{ whenever }~i ≤ j
 
+Specifying cumulativity
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The variance of the universe parameters for a cumulative inductive may be specified by the user.
```

Comment:
As I mentioned previously, I don't think `variance` is something any Coq user is expected to know. Even a random CS person may ignore it, it is very specific. Just look at the wikipedia page: https://en.wikipedia.org/wiki/Covariance_and_contravariance_(computer_science)

IMO it should be explained a little here.

Reply:
Variance is already in master, let's not block this PR over it. Feel free to open an issue asking for more doc.

Reply:
To prove my point here is a post from yesterday: https://news.ycombinator.com/item?id=24554849.

Reply:
We should probably have the rules somewhere in the core language section

Reply:
@ppedrot I am siding with @gares here. Not every Coq user is a PL researcher nor an HN reader.

Reply:
I don't have strong opinions about the best time to merge this PR, but it looks like it gets you to control a not very documented feature. I see that the manual talks about "universe variance" already, with apparently 0 explanation. I missed that change set, otherwise I would have complained back then.

If you need help to write the doc, we can try to find someone else. I'm not sure I qualify. To me variance is related to subtyping, and Coq's TT does not have subtyping... Oh wait...

Y combinator is name that says it all. @ppedrot go the math department and scream "lambda-calculus!" Then tell me how they look at you.

Reply:
https://coq.inria.fr/refman/language/cic.html#subtyping-rules

Reply:
Note that the word "variance" is not written once on the above page.

Reply:
Great, then this is where one should explain how the variance of an inductive in computed and where it plays a role in the rules. I guess it has something to do with `Gamma_P` but I don't really know. I did not even know that conversion could be called a proper subtyping relation (to me it was limited to a "subsort" relation, but I guess that now that it can cross some inductives we can talk about "subtypes").

Reply:
There's also https://coq.inria.fr/refman/language/core/inductive.html#theory-of-inductive-definitions and finally the cumulative inductive types are really only described in https://coq.inria.fr/refman/addendum/universe-polymorphism.html#cumulative-noncumulative. I wish for a consolidation of this material but I'm not sufficiently knowledgeable to really be able to take care of it or even make clear recommendations.

Reply:
I'm going trough the doc, and I see that these symbols (+, =, and *) are already mentioned. So I finally agree with @SkySkimmer that it is not for this PR to improve the situation. (but I hope someone does write down how variance is computed somewhere...)

Reply:
@gares if it does not already exist, would you mind opening an issue for the documentation of variance?